cmd/sync-tags/gomod.go

```diff
@@ -55,6 +55,10 @@ func updateGomodWithTaggedDependencies(tag string, depsRepo []string, semverTag
 		}
 
 		commit, commitTime, err := taggedCommitHashAndTime(dr, tag)
+		if err != nil {
+			// if tag has not been created on this volume, but pre-exists in Github, we have renamed that as origin/<tag>
+			commit, commitTime, err = taggedCommitHashAndTime(dr, "origin/"+tag)
```

Comment:
do we not get tags from origin because of this:

https://github.com/kubernetes/publishing-bot/blob/da9508d9d841255b64fc7ace6dc6010c5e225a5b/artifacts/scripts/construct.sh#L96-L97

I see a similar fallback in the godeps file, and slightly prefer a dedicated named function to do the fallback:

https://github.com/kubernetes/publishing-bot/blob/da9508d9d841255b64fc7ace6dc6010c5e225a5b/cmd/sync-tags/godeps.go#L132-L138

Reply:
` git fetch origin --no-tags --prune` does not download tags. We have another place (have to look for it) and prefix them with `origin/` in order to control which are new and which are not.

Reply:
wrap this fallback into a `localOrPublishedTaggedCommitHashAndTime` method, then lgtm

Reply:
done